daft/execution/udf.py

```diff
@@ -88,7 +88,7 @@ def __init__(self, project_expr: PyExpr, passthrough_exprs: list[PyExpr]) -> Non
         expr_projection = ExpressionsProjection(
             [Expression._from_pyexpr(expr) for expr in passthrough_exprs] + [Expression._from_pyexpr(project_expr)]
         )
-        expr_projection_bytes = daft.pickle.dumps(expr_projection)
+        expr_projection_bytes = daft.pickle.dumps((project_expr.name(), expr_projection))
```

Comment:
I realized this is the wrong name (this is the name for the expression, not the UDF), but since we're on a crunch. I fixed it in the next PR to avoid rerunning CI

Reply:
If I need to address any comments, will modify then